.aws/first.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
do we need to include `.aws` files? @cosmos/sre ?

Reply:
those are used by the current (temporary) node setup on EC2, until the real network boot on merge is ready